docs/component-dispatcher.md

```diff
@@ -0,0 +1,49 @@
+# Dispatcher Service
+
+The dispatcher service is a central component in the hub and spoke service collaboration. 
+It is responsible for delivering messages from one application to one or more target applications based on a tenant specific configuration.
+
+![Dispatcher Service Overview](https://github.com/openintegrationhub/openintegrationhub/blob/dispatcher/Assets/component_dispatcher_0.1.png)
+
+## Service
+### Entry Point
+
+The service is triggered by messages arriving through a SDF Adapter Component used in a Connector Flow. 
+
+### Tenant specific configuration
+
+Each tenant owns a configuration which specifies his applications and their synchronization behavior.
+The configuration contains information about:
+* which application is connected to another applications
+* full duplex/half duplex connections
+* leading system configuration
+
+Tenant specific configuration:
+
+| applicationId | domain | connected to OIH | inbound | outbound |
+| ------------- | ------------- | ------------- | ------------- | ------------- |
+| crm1 | customer | true | true | true |
+| contactmanagement2 | customer | true | true | true |
+| cms1 | product | true | false | true |
+
+
+
+For each received message, the configuration is evaluated to determin how to process the message. 
+Message characteristics relevant for the dispatcher service:
+* tenant id
+* source application
+* source operation
+
```

Comment:
What about the resource as a needed characteristic? Maybe a tenant only wants an application to operate within a specific domain?

Example: `Resource`.Operations : `Contact`.Create

Reply:
source domain added